packages/core/protocols/src/proto/dxos/iframe.proto

```diff
@@ -43,11 +42,10 @@ service AppService {
 
 enum ShellLayout {
   DEFAULT = 0;
-  AUTH = 1;
-  CURRENT_SPACE = 2;
-  SPACE_LIST = 3;
+  INITIALIZE_IDENTITY = 1;
+  HALO_INVITATIONS = 2;
```

Comment:
DEVICE vs SPACE?

Reply:
I've been considering that as well but figured I'd align with the name of the invitation service for now